temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt

```diff
@@ -501,7 +501,7 @@ fun KeypadCircleButton(
 ) {
     Text(
         modifier = circleButtonModifier(size = btnSize, onClick = onClick)
-            .padding(top = 10.dp)
+            .padding(13.dp)
```

Comment:
Can you make the padding `top` only? Otherwise we add padding on all 4 sides which risks the number on the button to become cut.